src/LSODA.cpp

```diff
@@ -315,7 +315,7 @@ void LSODA::lsoda(LSODA_ODE_SYSTEM_TYPE f, const size_t neq, vector<double> &y,
         {
           // cerr << "[lsoda] tout = " << tout << " behind t = " << *t
           //      << ". integration direction is given by " << h0 << endl;
-          REprintf("[lsoda] tout = %d behind t = %d. integration direction is given by %d.\n", 
+          REprintf("[lsoda] tout = %.f behind t = %f. integration direction is given by %f.\n", 
```

Comment:
Is the `.` in front of  the first `f` unintended?  (I think that will chop off everything after the decimal.)

Reply:
Thanks; didn't pick that up.